assets/js/pages/Auth.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+
+const Auth = () => {
+  const navigate = useNavigate();
+  const redirectToHomePage = () => navigate("/");
+
+  const fetchToken = (google_response) => {
+    const request: RequestInit = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify(google_response),
+    };
+
+    fetch(window.location.origin + "/auth/login", request)
+      .then((resp) => resp.json())
+      .then((data) => localStorage.setItem("jwt", data["token"]));
+  };
+
+  const fetchTokenAndRedirect = (response) => {
+    fetchToken(response);
+    redirectToHomePage();
+  };
+
+  useEffect(() => {
+    google.accounts.id.initialize({
+      client_id: "1003639280735-i6pl1d6m7f70m4ml66hgbno54qdj4a7o.apps.googleusercontent.com",
```

Comment:
In this or in the future PR it should probably be moved to some config or get from an environment variable.